lerobot/configs/train.py

```diff
@@ -34,11 +34,10 @@
 
 @dataclass
 class TrainPipelineConfig(HubMixin):
-    dataset: DatasetConfig
+    dataset: DatasetConfig | None = None  # NOTE: In RL, we don't need a dataset
```

Comment:
We need to discuss whether we should keep this addition. Discussions with @imstevenpmwork  have highlighted potential side effects. Please raise any concerns you have regarding this part.

Reply:
You don’t necessarily need an offline dataset with RL, you can directly cold-start your training without any pre-collected data. Therefore, dataset.repo_id isn’t mandatory for RL

Reply:
COuld we add comment `offline dataset`

Reply:
Done